evm_trace/base.py

```diff
@@ -194,3 +194,56 @@ def _create_node_from_call(
     # TODO: Handle "execution halted" vs. gas limit reached
 
     return node
+
+
+def create_node_from_parity_trace(item, traces):
```

Comment:
Would it be beneficial to have Pydantic models for the trace objects here too?

Reply:
It cleans things up a little from a typing perspective. could also use pydantic dataclasses